src/modules/app/components/side-nav/side-nav.jsx

```diff
@@ -122,6 +123,26 @@ export default class SideNav extends Component {
             </li>
           }
         </ul>
+        {isLogged && isMobile &&
+        <div className={Styles.SideName__space} />
```

Comment:
No need for this — you can apply the flex directly to the `.SideNav__nav`, which should result in the same outcome.